src/constants/labels.json

```diff
@@ -109,13 +109,21 @@
       "message3": "Wörter richtig. Und du?"
     }
   },
+  "feedback": {
+    "sendFeedback": "Senden",
+    "question": "Hast du Feedback oder einen Wortvorschlag?",
+    "mailPlaceholder": "Deine Email für Fragen(optional)",
```

Comment:
space between Fragen and (optional) would be nice, although it is missing in the designs.